import_flow_events.py

```diff
@@ -253,23 +271,6 @@
       AND flow_metadata.begin_time <= '{day}'::DATE + '1 day'::INTERVAL;
 """
 
-Q_INSERT_EVENTS = """
```

Comment:
I like to keep these definitions in invocation order to make reading them more logical, since they're a bit distant from the point of usage.